OCM/Common/Components/Content List/ContentListView.swift

```diff
@@ -57,6 +62,8 @@ class ContentListView: UIView {
     fileprivate var cellSelected: UIView?
     fileprivate var cellFrameSuperview: CGRect?
     fileprivate var refresher: UIRefreshControl?
+    fileprivate var paginationControl: ImageActivityIndicator?
```

Comment:
shouldn't this be something like paginationActivityIndicator? Control is very generic.